src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -76,12 +86,12 @@ def addon_selected?(_addon)
       # @return [Yast::Term] the main UI dialog term
       def content
         VBox(
-          VStretch(),
           Left(Heading(heading)),
+          Left(CheckBox(Id(:filter_beta), Opt(:notify),
```

Comment:
@lslezak I see. Your points make sense but I think they are relatively minor at this point so I will not rush to fix them.